test/message/promise_fast_reject.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// We should always have the stacktrace of the oldest rejection.
```

Comment:
I'm not sure this is behavior we can rely on - if V8 decides to GC `'Two'` first that's entirely in its right.

Reply:
I somewhat agree. It seems like we can rely on it for now though. Would you still want this to be changed?